src/app/mail/mail-settings/mail-settings.component.ts

```diff
@@ -121,6 +125,18 @@ export class MailSettingsComponent implements OnInit, OnDestroy {
       'contact_email': ['', [Validators.pattern(VALID_EMAIL_REGEX)]],
       'password': ['', [Validators.required]]
     });
+    this.route.params.subscribe(
+      params => {
+        this.forwardingQueryParams = params['id'];
```

Comment:
the type of variable `forwardingQueryParams ` should be string, not any.
the variable `forwardingQueryParams ` should be named properly.